porfolio.html

```diff
@@ -16,7 +16,41 @@
             <p class="text">John Doe</p>
         </div>
     </div>
- 
+    <div class="headline">
+        <div class="primary-text">Hey There. <br>
+            I’m Fuad
+        </div>
+        <div class="supporting-text">I’am a Software Developer
+        </div>
+        <div class="description">
```

Comment:
Kindly use a `<p>` tag. This way will be semantically correct and the screen readers and other technologies will thank you